docs/content/en/docs/reference/clusterspec/vsphere.md

```diff
@@ -305,7 +305,7 @@ The default is generating a key in your `$(pwd)/<cluster-name>` folder when not
 ### template (optional)
 The VM template to use for your EKS Anywhere cluster. This template was created when you
 [imported the OVA file into vSphere]({{< relref "../vsphere/vsphere-ovas.md" >}}).
-This is a required field if you are using Ubuntu-based or RHEL-based OVAs.
+This is a required field if you are using Ubuntu-based or Redhat-based OVAs.
```

Comment:
I've been using "RHEL-based". Is there a reason you said "Redhat-based"?  I think Red Hat might complain about having their name used that way. Though not a perfect solution, RHEL is pretty commonly known and better than writing out the whole "Red Hat Enterprise Linux-based".

Reply:
I dont mind either to be honest.